sor/src/main/java/com/bazaarvoice/emodb/sor/db/DeltaIterator.java

```diff
@@ -194,11 +208,26 @@ private ByteBuffer stitchContent(int contentSize) {
         return content;
     }
 
-    // transforms delta from the provided type to return type
-    abstract protected T convertDelta(R delta);
+    protected static class BlockedDelta {
+        private final int _numBlocks;
+        private final ByteBuffer _content;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_content" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)